orc8r/protos/bootstrapper.proto

```diff
@@ -71,3 +73,83 @@ service Bootstrapper {
   // Returns signed certificate.
   rpc RequestSign (Response) returns (Certificate) {}
 }
+
+// --------------------------------------------------------------------------
+// Protos for CloudRegistration and Registration servicer
+// --------------------------------------------------------------------------
+service CloudRegistration {
+  // GetToken saves and sends out a token related to input:networkID and input:logicalID
+  // It will refresh the token if input:refresh is true or token is expired
+  rpc GetToken(GetTokenRequest) returns (GetTokenResponse) {}
```

Comment:
Adding comments for the service definition can be helpful in explaining how this is intended to be used at a high level. As a new developer, I would wonder how the token/nonce is used, but it's not clear where to look after seeing this service definition.

Reply:
Added a bit of a written diagram -- bloats the code a bit